tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
+    "lib": ["es2015", "dom"],
```

Comment:
We should be switching these to `es2018` to snap with the latest node.js support via https://node.green